src/components/SecondaryPanes/Frames/Group.css

```diff
@@ -14,24 +14,33 @@
   direction: ltr;
 }
 
-.frames [role="list"] .frames-group.expanded .group,
-.frames [role="list"] .frames-group.expanded .group .location {
+.frames-group.expanded .group,
+.frames-group.expanded .group .location {
   color: var(--theme-highlight-blue);
 }
 
-.frames [role="list"] .frames-group .frames-list [role="listitem"] {
-  padding-left: 30px;
+.frames-group .frames-list [role="listitem"] {
+  padding-inline-start: 32px;
 }
 
-.frames [role="list"] .frames-group .frames-list {
+.frames-group .frames-list {
   border-top: 1px solid var(--theme-splitter-color);
   border-bottom: 1px solid var(--theme-splitter-color);
 }
 
-.frames [role="list"] .frames-group.expanded .badge {
+.frames-group.expanded .badge {
   color: var(--theme-highlight-blue);
 }
 
-.group-description-name {
-  padding-left: 5px;
+.frames-group .img.arrow {
+  margin-inline-start: -1px;
+}
+
+.frames-group .img.annotation-logo {
```

Comment:
I think we can have `.annotation-logo` not in `.frames-group`, for example when you have only 1 library frame. Is this change (and the related one in `Frames.css`) impact those?

Reply:
It looks like you are rolling your own styles in `devtools/client/shared/components/SmartTrace.css`?
So this probably won't affect Console. But I'll move it back to Frames.css (scoped with `.frames` instead of `.frames-group`) to be on the safe side.